src/components/footer.js

```diff
@@ -80,9 +81,8 @@ function Footer({ stripes }) {
                   <FaGithub
                     size={16}
                     css={{
-                      marginRight: '0.25rem',
-                      position: 'relative',
-                      top: 2,
+                      marginRight: '0.5rem',
+                      verticalAlign: 'sub',
```

Comment:
👍

Reply:
It does what we want it to, but not sure why ;) 🤷‍♂️

Reply:
It's _way_ cleaner than what I had 😆

Reply:
Hate when I have to resort to those hacks!